react-graphql-apollo-tutorial.md

```diff
@@ -1021,9 +1023,9 @@ const RepositoryItem = ({
 );
 {{< /highlight >}}
 
-Note: The surrounding div element of the Mutation component is already there for other mutations that you will implement in the exercises of this section.
+Note: The div element surrounding the Mutation component is there for other mutations you will implement in this section.
 
-The `id` for each repository should be available due to previous query result. It has to be used as a variable for the mutation to identify the repository. Furthermore, the Mutation component is used in a similar way as the Query component, because it implements the render prop pattern too. But the first argument is different here: It is the mutation as a function instead of the mutation result this time. Hence you can use this function to trigger the mutation before expecting a result. Later you will see how to retrieve the mutation result. For now, the mutating function can be used in a button element. Or in this case already in a Button component.
+The `id` for each repository should be available due to previous query result. It has to be used as a variable for the mutation to identify the repository. The Mutation component is used in a way like theQuery component, because it implements the render prop pattern as well. The first argument is different, though, as it a mutation as a function instead of the mutation result. Use this function to trigger the mutation before expecting a result. Later, you will see how to retrieve the mutation result; for now, the mutating function can be used in a button element. In this case, it is already in a Button component:
```

Comment:
"in a way like theQuery component, because" to ""in a way like the Query component, because""

Reply:
"as it a mutation as a function instead" -> "as it is the mutation function itself instead"